haiku-games/hexvexed/hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A tile puzzle game for Haiku"
+DESCRIPTION=""HexVexed is a maddening addictive and easy to play puzzle game: \
+Arrange hexagonal tiles into a grid so that the numbers and colours match \
+where they touch. This game is a version DarkWyrm's BeVexed, but instead \
+of squares, the tiles a hexagonal shaped. It's 1.5x the fun!"
+
```

Comment:
Oops, I messed this one.
No need for a blank line between DESCRIPTION and HOMEPAGE, as described in our [HaikuPorter ghuidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering).